collect_app/src/main/java/org/odk/collect/android/widgets/BaseImageWidget.java

```diff
@@ -30,7 +30,6 @@
 import android.widget.Toast;
```

Comment:
Deprecated getInstanceFolder() replaced by calls to field instance. 
Adds null checks in getFile(), addCurrentImageToLayout().